autoware_map_msgs/srv/LoadDifferentialPointCloudMap.srv

```diff
@@ -0,0 +1,13 @@
+# Area query for map loading
+# (PCD grids that overlaps with the area is going to be loaded)
+AreaInfo area
+
+# The IDs of PCD maps that the node already has
+string[] already_loaded_ids
+
+---
+# Newly loaded PCD maps with ID
+PointCloudMapWithID[] loaded_pcds
```

Comment:
```suggestion
PointCloudMapWithID[] new_pointcloud_with_ids
```

Following up from 
https://github.com/autowarefoundation/autoware_msgs/pull/39#discussion_r981953600

Name PCD is not unfamiliar but your message is made of PointCloud2 ros message, which is very different from the [PCD Format](https://pointclouds.org/documentation/tutorials/pcd_file_format.html).

Reply:
Thanks! Resolved https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/27a8ea6714aeac6be4f1b394e994aad73fdfa167